### PR TITLE
Base Project

### LINKED ISSUE #1: Base Project
Please add base project

### PR DESCRIPTION
**Pull request by samuelloza**
I add base project

### CODE REVIEW

**Review by MirkoFer-622 — CHANGES_REQUESTED** [review]
I think that the directory structure below is ambiguos (a directory named api under gui?)
imagen

I think "salesforce" directory should be under "org.fundacionjala" (src/main/java/org/fundacionjala/salesforce) directories and into "salesforce" should be both "gui" and "api" directories with their subdirectories.

File: .gitignore [hunk 1]
```diff
@@ -1,14 +1,38 @@
...
-# Cache of project
-.gradletasknamecache
+.allure
 
-# # Work around https://youtrack.jetbrains.com/issue/IDEA-116898
-# gradle/wrapper/gradle-wrapper.properties
+bin/
+.classpath
+.project
+.settings/
```
Comment (MirkoFer-622) [inline .gitignore:38]: please add ".idea" to this file

Comment (samuelloza) [reply .gitignore:38]: The ".idea"  is on the line 33

**Comment by samuelloza** [comment]
> I think that the directory structure below is ambiguos (a directory named api under gui?)
> imagen
> 
> I think "salesforce" directory should be under "org.fundacionjala" (src/main/java/org/fundacionjala/salesforce) directories and into "salesforce" should be both "gui" and "api" directories with their subdirectories.

uh sorry, I changing to the new structure, 
but I don't if this structure is correct ? @MirkoFer-622 do you believe is correct ?

image

**Comment by MirkoFer-622** [comment]
> 
> 
> > I think that the directory structure below is ambiguos (a directory named api under gui?)
> > imagen
> > I think "salesforce" directory should be under "org.fundacionjala" (src/main/java/org/fundacionjala/salesforce) directories and into "salesforce" should be both "gui" and "api" directories with their subdirectories.
> 
> uh sorry, I changing to the new structure,
> but I don't if this structure is correct ? @MirkoFer-622 do you believe is correct ?
> 
> image

I think ui and api should be in the same level, under "salesforce" directory, and salesforce should be under org.fundacionjala, plese

File: .gitignore (hunk 1 above)
Comment (MirkoFer-622) [reply .gitignore:38]: Yes, I could review it, it's good.

**Review by MirkoFer-622 — APPROVED** [review]

**Comment by MirkoFer-622** [comment]
LGTM!

**Pull request merged**